### PR TITLE
streammusic update livecheck url, zap, bump to 1.3.4

### PR DESCRIPTION
**Pull request by jibwf**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [1] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [1] `brew audit --cask --online <cask>` is error-free.
- [1] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [ ] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [ ] `brew audit --cask --new <cask>` worked successfully.
- [ ] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [ ] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/s/streammusic.rb [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
   name "StreamMusic"
   desc "Music client compatible with self-hosted music services"
   homepage "https://www.aqzscn.cn/"
 
   livecheck do
-    url "https://music.aqzscn.cn/docs/versions/latest/"
-    regex(/href=.*v?(\d+(?:\.\d+)+(?:\.\d+)+)\.dmg/i)
+    url :url
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
   end
```
Comment (bevanjkay) [inline Casks/s/streammusic.rb:15]:
```suggestion
```

The `livecheck` should not be necessary with the change to a GitHub `url` as the default `git` livecheck strategy will automatically be inferred from the url.

**Review by bevanjkay — COMMENTED** [review]
Do you have a reference for the updated source? I can't find any link to the GitHub repository from the website.

File: Casks/s/streammusic.rb (hunk 1 above)
Comment (jibwf) [reply Casks/s/streammusic.rb:15]: image

Thanks for comments on livecheck part. 

The GitHub link is at the top right corner of the homepage.

Comment (bevanjkay) [reply Casks/s/streammusic.rb:15]: I did now find a reference in the docs. If you can remove the `livecheck` as suggested above we should be good to go here.

Comment (jibwf) [reply Casks/s/streammusic.rb]: removed livecheck part as suggested.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**